pkg/pillar/agentlog/agentlog.go

```diff
@@ -292,6 +337,46 @@ func listenDebug(log *base.LogObject, stacksDumpFileName, memDumpFileName string
 			return
 		}
 	}))
+	mux.Handle("/memory-monitor/psi-collector/start", http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
It's out of the scope of this PR, but how about we move from http.Mux to chi router, which supports middleware and you can separate POST and GET methods as well? We use it in metadata server

Reply:
Mmm... I don't mind. Feel free to implement or fire a ticket so as not to forget it =)